edweiss-app/components/courses/AssignmentDisplay.tsx

```diff
@@ -55,127 +33,68 @@ export const testIDs = {
     assignmentDate: 'assignment-date'
 };
 
-
-// Hardcoded paths
-const hardcodedPathToQuizzes = "courses/placeholderCourseId/quizzes";
-
-// Hardcoded IDs
-const hardcodedQuizId = "TestQuizId";
-const hardcodedCourseId = "placeholderCourseId";
-
-
-export type AssignmentWithColor = AssignmentBase & { color: string; };
-
-
-const AssignmentDisplay: ReactComponent<{ item: AssignmentWithColor, index: number, backgroundColor: string, swipeBackgroundColor: string, defaultColor: string, swipeableRefs: React.MutableRefObject<(Swipeable | null)[]>; }> = ({ item, index, backgroundColor, swipeBackgroundColor, defaultColor, swipeableRefs }) => {
-
+export type AssignmentWithColor = Assignment & { color: Color; };
+
+
+/**
+ * AssignmentDisplay Component
+ * 
+ * This component is responsible for displaying an assignment in the course page.
```

Comment:
I think that you can delete this comments.

Reply:
I prefer to let it since It could be useful to understand the use of the component